modules/candidate_list/php/candidate_list.class.inc

```diff
@@ -74,8 +74,9 @@ class Candidate_List extends \DataFrameworkMenu
         \Module::factory('candidate_parameters');
 
         // create user object
-        $user   = \User::singleton();
-        $config = \NDB_Config::singleton();
+        $factory = \NDB_Factory::singleton();
+        $user    = $factory->user();
+        $config  = \NDB_Config::singleton();
```

Comment:
Hi @laemtl, you can also do `$factory->config();` for accessing the config. @johnsaigle showed me that.

Reply:
Hello @maltheism,

Thanks for pointing this out! 
I noticed the config method in /NDB_Factory.class.inc and was curious why it was not referenced anywhere in the unit testing guide. It's definitely a good idea to uniformize everything at once. I will review all the submitted modules.

Reply:
Thanks for the PR @laemtl, this is a great improvement. If you're up for it amending the Unit Test Guide would be really helpful also. If not I'll make a separate issue for it.